steno/src/view/components/top_bar.cpp

```diff
@@ -92,8 +105,18 @@ void TopBar::onRecorderStateChanged(const QMediaRecorder::State& state)
             m_ui->recordButton->setText("Stop recording");
             break;
         case QMediaRecorder::State::StoppedState:
+        {
             m_ui->recordButton->setText("Start recording");
+            const auto transcriptionConfig = m_config->transcriptionConfig();
+            const bool isTranscriptionEnabled =
+                transcriptionConfig->value(Model::TranscriptionConfig::AUTOMATIC_TRANSCRIPTION).toBool();
+            if (isTranscriptionEnabled)
+            {
+                m_transcription->transcribe(QCoreApplication::applicationDirPath() +
```

Comment:
Should we keep this until we can make the call on our recorded files?

Reply:
I already finished the real implementation for the next pull request.